src/components/Book.js

```diff
@@ -31,7 +26,7 @@ const Book = (obj) => {
         <div className="functions">
           <p>Comments</p>
           <hr />
-          <button style={button} type="button" onClick={() => RemoveBtn(id)}>Remove</button>
+          <button className="button" type="button" onClick={() => RemoveBtn(id)}>Remove</button>
```

Comment:
- [ ] The the design there is a circle present to represent the completion percentage. Please make it in your design too. 

Your Design: 

![image](https://user-images.githubusercontent.com/58504655/192026837-0a49af33-0ce6-434a-bf08-a76e82c06d3e.png)

Original 

![image](https://user-images.githubusercontent.com/58504655/192026897-7dc63be6-fa4d-4a5a-a328-d951681c64f2.png)